src/frontend/components/SøknadsSteg/2-VelgBarn/LeggTilBarn/useLeggTilBarn.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+
+import { FormattedMessage } from 'react-intl';
+
+import { ESvar } from '@navikt/familie-form-elements';
+import { feil, ISkjema, ok, useFelt, useSkjema, Valideringsstatus } from '@navikt/familie-skjema';
+import { idnr } from '@navikt/fnrvalidator';
+
+import { IBarnNy } from '../../../../typer/person';
+import SpråkTekst from '../../../Felleskomponenter/SpråkTekst/SpråkTekst';
+import { ESvarMedUbesvart } from '../../1-OmDeg/useOmdeg';
+
+export interface ILeggTilBarnTyper extends Omit<IBarnNy, 'borMedSøker' | 'alder'> {
+    borMedSøker: ESvarMedUbesvart;
+    erFødt: ESvarMedUbesvart;
+    navnUbestemt: ESvar;
+    ingenIdent: ESvar;
+}
+
+export const useLeggTilBarn = (): {
+    skjema: ISkjema<ILeggTilBarnTyper, string>;
+    validerFelterOgVisFeilmelding: () => boolean;
+    valideringErOk: () => boolean;
+    nullstillSkjema: () => void;
+} => {
+    const erFødt = useFelt<ESvarMedUbesvart>({
+        verdi: undefined,
+        valideringsfunksjon: felt => {
+            return felt.verdi === ESvar.JA
+                ? ok(felt)
+                : feil(felt, <SpråkTekst id={'Du kan ikke søke for ufødte barn'} />);
```

Comment:
Her skal det kanskje være en id i stedet for klartekst? 😊

Reply:
Kanskje :)